<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="MyResults">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>My Results</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>thyroid.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="TakeAssessment">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>265</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Take a self-assessment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 127);
font: 87 18pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>THY-SYS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Logout">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>455</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(188, 188, 188);</string>
   </property>
   <property name="text">
    <string>© Icon from flaticon.com</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>371</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0.78486, y2:0.869, stop:0.364486 rgba(170, 85, 255, 255), stop:1 rgba(85, 170, 255, 255));
border-radius:50px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>ⓘ</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>MyResults</zorder>
  <zorder>label</zorder>
  <zorder>TakeAssessment</zorder>
  <zorder>label_2</zorder>
  <zorder>Logout</zorder>
  <zorder>label_3</zorder>
  <zorder>info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
